### PR TITLE
Support Monitoring DynamoDB.

### PR DESCRIPTION
**Pull request by yswdqz**
### <Feature description>
- [ ] If this is non-trivial feature, paste the links/URLs to the design doc.
- [x] Update the documentation to include this new feature.
- [x] Tests(including UT, IT, E2E) are added to verify the new feature.
- [x] If it's UI related, attach the screenshots below.

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

image
image
image
image

### CODE REVIEW

File: docs/en/setup/backend/backend-aws-dynamodb-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+### Data flow
+1. Amazon CloudWatch fetches metrics from DynamoDB and pushes metrics to SkyWalking OAP Server via Amazon Kinesis data firehose.
+2. The SkyWalking OAP Server parses the expression with [MAL](../../concepts-and-designs/mal.md) to filter/calculate/aggregate and store the results.
+
+### Set up
+1. Enable [AWS CloudWatch](https://aws.amazon.com/cn/cloudwatch/)
+2. Create [Amazon Kinesis Data Filehose](https://aws.amazon.com/cn/kinesis/data-firehose/), set source to `Direct PUT`, set destination to  `HTTP Endpoint`, and set `HTTP EndPoint url` to `aws-firehose-receiver`'s port (refer to [aws-firehose-receiver](aws-firehose-receiver.md))
+
+   Note that AWS requires that the `HTTP Endpoint URL` must be through HTTPS listening at 443, therefore need to load the certificate in [aws-firehose-receiver](aws-firehose-receiver.md) and set the port to 443.
+   Or, you can use another gateway to accept the requests and route them to `aws-filehose-receiver`.
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-aws-dynamodb-monitoring.md:13]: I think we need a proxy as always. And we don't need to mention this part again, as `firehose receiver` doc includes this before.

File: docs/en/setup/backend/backend-aws-dynamodb-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+3. Create a [metric stream](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Metric-Streams.html), set namespace to DynanoDB, and set `Kinesis Data Firehose` to the firehose you just created.
+4. Config [aws-firehose-receiver](aws-firehose-receiver.md) to receive data.
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-aws-dynamodb-monitoring.md:15]: You could refer s3 document, the most important part of this doc is for pointing to AWS docs about how to setup firehose for DynamoDB, and forward to OAP.

**Comment by wu-sheng** [comment]
Should we put `inspections` as the tab0? Usually we want to see general metrics first, then to check for table by table.

**Comment by yswdqz** [comment]
Got it.

**Comment by yswdqz** [comment]
Now it is ok.
image

**Review by wu-sheng — APPROVED** [review]
LGTM

**Review by wankai123 — APPROVED** [review]

**Pull request merged**